pkg/features/antrea_features.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package features
+
+import (
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/component-base/featuregate"
+)
+
+const (
+	// Every feature gate should add method here following this template:
+	//
+	// alpha: vX.Y
+	// beta: vX.Y
+	// MyFeature featuregate.Feature = "MyFeature"
+
+	// alpha: v0.8
+	// Allows to apply cluster-wide Network Policies. Users can group Policies together in Categories and define their
```

Comment:
Maybe it is not from you, but for "Users can group Policies together in Categories and define their order of execution", we do not have categories yet right? Maybe remove this sentence.

And "Network Policies" to "NetworkPolicies"?

Reply:
done